### ISSUE TITLE
missing option for proper formatting import statements with multiple members

### ISSUE THREAD

**Issue by axten** [eslint/eslint#11508]
**What rule do you want to change?**
object-property-newline or object-curly-newline

**Does this change cause the rule to produce more or fewer warnings?**
more warnings

**How will the change be implemented? (New option, new default behavior, etc.)?**
I thinks the fix could be new value for option allowAllPropertiesOnSameLine in rule object-property-newline but I'm not sure:
something like: allowAllPropertiesOnSameLine: 'singleLine'

this value should ensure that allowAllPropertiesOnSameLine behaviour is only applied when the object declaration is completly on a single line.

**Please provide some example code that this change will affect:**

the rules are currently cofigured this way:
```js
    'object-curly-newline': ['error', {
      ObjectExpression: { minProperties: 4, multiline: true, consistent: true },
      ObjectPattern: { minProperties: 4, multiline: true, consistent: true },
      ImportDeclaration: { minProperties: 4, multiline: true, consistent: true },
      ExportDeclaration: { minProperties: 4, multiline: true, consistent: true },
    }],

    'object-property-newline': ['error', {
      allowAllPropertiesOnSameLine: true,
    }],
```

Examlpe
```js
import { A, B, C } from 'module';
import { A, B, C, D } from 'module';
```

**What does the rule currently do for this code?**
current output:
```js
import { A, B, C } from 'module';
import { 
    A, B, C, D,
} from 'module';
```

**What will the rule do after it's changed?**
Expeted output:
```js
import { A, B, C } from 'module';
import { 
    A, 
    B, 
    C, 
    D,
} from 'module';
```

_I know that indention and trailings commas are handled by other rules_

**Are you willing to submit a pull request to implement this change?**
I'm not able atm.

**Comment by axten** [comment]
great, and now I have to copy paste and open a new one?
